src/lib/components/PageHeader.svelte

```diff
@@ -0,0 +1,43 @@
+<script lang="ts">
+  export let testId: string | undefined = undefined;
+
+  let hasHeader: boolean;
+  $: hasHeader =
+    $$slots.topLeft !== undefined || $$slots.topRight !== undefined;
+</script>
+
+<div class="container" data-tid={testId}>
+  {#if hasHeader}
+    <div class="header">
+      <slot name="topLeft" />
+      <slot name="topRight" />
+    </div>
+  {/if}
+  <div class="content">
+    <slot name="title" />
+    <slot name="subTitle" />
```

Comment:
Is "subtitle" a word? should the slot name be lower case then?

Reply:
yes, it is a word. Thanks